### PR TITLE
[12.0][IMP] quality_control: Added triggers on product.product views

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT28453

please @pedrobaeza @joao-p-marques review this

### CODE REVIEW

File: quality_control/README.rst [hunk 1]
```diff
@@ -14,13 +14,13 @@ Quality control
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmanufacture-lightgray.png?logo=github
-    :target: https://github.com/OCA/manufacture/tree/12.0/quality_control
+    :target: https://github.com/OCA/manufacture/tree/13.0/quality_control
```
Comment (pedrobaeza) [inline quality_control/README.rst]: You have regenerated the README with an incorrect version.

**Comment by CarlosRoca13** [comment]
Ups! Changed @pedrobaeza

**Comment by pedrobaeza** [comment]
The rest is OK.

**Review by pedrobaeza — APPROVED** [review]

File: quality_control/views/product_template_view.xml [hunk 2]
```diff
@@ -8,9 +8,9 @@
 <odoo>
 
     <record model="ir.ui.view" id="product_template_qc_form_view">
-        <field name="name">product.template.qc</field>
+        <field name="name">product.template.common.qc</field>
         <field name="model">product.template</field>
-        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
```
Comment (joao-p-marques) [inline quality_control/views/product_template_view.xml:13]: I am unable to test this. Shouldn't this be added both in `product.product_template_only_form_view` (templates) **and** in `product.product_normal_form_view` (variants). I can't see the button in any of them. I've checked the group permissions and they seem correct.

**Review by joao-p-marques — CHANGES_REQUESTED** [review]

File: quality_control/views/product_template_view.xml (hunk 2 above)
Comment (CarlosRoca13) [reply quality_control/views/product_template_view.xml:13]: I can see it in both... See next images from runbot
image
image

Comment (joao-p-marques) [reply quality_control/views/product_template_view.xml:13]: Ok, I can see those. I thought this referred to a different element, MB :+1:

**Review by joao-p-marques — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please check v13+ branches

**Pull request merged**